wordpress-mobile/gutenberg-mobile
# Video block - don't force full screen on iOS

Updates the video player to not force fullscreen on the iOS video block.

I also added a prop that allows the explicit setting of fullscreen mode when pressing play for iOS and Android. This can be helpful in the future for the Cover block.

GB PR: https://github.com/WordPress/gutenberg/pull/20436

To test:
 - Run the demo app
 - on iOS check that the video on the demo doesn't go to fullscreen when pressing play
 - check if no layout issues happen
 - on Android check, if the video block still open on an external application
 - check if no layout issues happen

PR submission checklist:
